template/en/default/bug/new_bug.html.tmpl

```diff
@@ -0,0 +1,49 @@
+[%# The contents of this file are subject to the Mozilla Public
+  # License Version 1.1 (the "License"); you may not use this file
+  # except in compliance with the License. You may obtain a copy of
+  # the License at http://www.mozilla.org/MPL/
+  #
+  # Software distributed under the License is distributed on an "AS
+  # IS" basis, WITHOUT WARRANTY OF ANY KIND, either express or
+  # implied. See the License for the specific language governing
+  # rights and limitations under the License.
+  #
+  # The Original Code is the Bugzilla Bug Tracking System.
+  #
+  # The Initial Developer of the Original Code is Netscape Communications
+  # Corporation. Portions created by Netscape are
+  # Copyright (C) 1998 Netscape Communications Corporation. All
+  # Rights Reserved.
+  #
+  # Contributor(s): Sebastin Santy <sebastinssanty@gmail.com>
+  #
+  #%]
```

Comment:
This is the old (MPL v1 header) -- it should use the new (MPL v2) header -- I think this is the case in Bugzilla/CGI/ContentSecurityPolicy.pm for instance.